{"name": "test_user_add", "status": "failed", "statusDetails": {"message": "AssertionError: assert <selenium.webdriver.remote.webelement.WebElement (session=\"6fcb26a0b1b02e4a5c477542b76602f3\", element=\"9f22566f-3a09-4d6d-82f6-76a9de503563\")> == '添加用户成功'", "trace": "self = <test_case.test_002_user_add.Test_User_add object at 0x0000000003927EB0>\n\n    @allure.story(\"正常添加用户\")\n    @allure.severity(allure.severity_level.NORMAL)\n    def test_user_add(self):\n        \"\"\"正常添加用户\"\"\"\n        Logger().info(\"user add success\")\n        try:\n            self.user_add.user_add(username, password, email)\n        except NoSuchElementException:\n            Logger().error(\"user_add no find element\")\n        text = User_Add_Page_Locator().user_add_succee()\n>       assert text == \"添加用户成功\"\nE       assert <selenium.webdriver.remote.webelement.WebElement (session=\"6fcb26a0b1b02e4a5c477542b76602f3\", element=\"9f22566f-3a09-4d6d-82f6-76a9de503563\")> == '添加用户成功'\n\ntest_case\\test_002_user_add.py:43: AssertionError"}, "description": "正常添加用户", "attachments": [{"name": "log", "source": "3fd503fd-d4f9-4c4d-a177-9f3eb0bb9580-attachment.txt", "type": "text/plain"}], "start": 1663559799027, "stop": 1663559814260, "uuid": "bbd1f7e8-09c9-4962-bcc9-7fa628920ef5", "historyId": "84039d43224d4034e83312c8ca354a99", "testCaseId": "36b76c837d963a84b27ec03de67a9f4c", "fullName": "test_case.test_002_user_add.Test_User_add#test_user_add", "labels": [{"name": "story", "value": "正常添加用户"}, {"name": "suite", "value": "用户列表添加用户测试用例"}, {"name": "severity", "value": "normal"}, {"name": "parentSuite", "value": "test_case"}, {"name": "subSuite", "value": "Test_User_add"}, {"name": "host", "value": "PC-201903070025"}, {"name": "thread", "value": "5896-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.test_002_user_add"}]}